charts/rollups-node/README.md

```diff
@@ -160,4 +162,4 @@ Cartesi will release a new chart updating its containers if a new version of the
 This chart allows you to set your custom affinity. Find more information about Pod's affinity in the [Kubernetes documentation](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity).
 
 ----------------------------------------------
-Autogenerated from chart metadata using [helm-docs v1.12.0](https://github.com/norwoodj/helm-docs/releases/v1.12.0)
\ No newline at end of file
+Autogenerated from chart metadata using [helm-docs v1.11.0](https://github.com/norwoodj/helm-docs/releases/v1.11.0)
```

Comment:
You could update your local copy of helm-docs, so we don't have this version rollback in the footer.

Reply:
It's weird because I used docker.